Bio/SeqUtils/MeltingTemp.py

```diff
@@ -498,15 +534,17 @@ def salt_correction(Na=0, K=0, Tris=0, Mg=0, dNTPs=0, method=1, seq=None):
 
     """
     if method in (5, 6, 7) and not seq:
-        raise ValueError("sequence is missing (is needed to calculate " +
-                         "GC content or sequence length).")
+        raise ValueError(
+            "sequence is missing (is needed to calculate "
+            + "GC content or sequence length)."
```

Comment:
I think the `+` in the second line can be ommited

Reply:
Yes - that would be nicer.

Reply:
done.